test/GRAPHFILES

```diff
@@ -65,6 +65,7 @@ performance/compiler/bradc/AllCompTime.graph
 memleaks.graph
 studies/jacobi/jacobi.graph
 performance/elliot/no-op.graph
+performance/bharshbarg/forall-dom-range.graph
```

Comment:
I think this should also go under the "Array/Domain performance" suite. And maybe move it to after the "arrays/diten/time_array_vs_ddata.graph" graph in this suite, I think that's a better logical place for it, so you see array graphs next to each other